### ISSUE TITLE
Make not installed by default

### ISSUE THREAD

**Issue by ianaz** [nektos/act#206]
Hi guys, thanks for your efforts!
Actions in Github seem to have make installed by default while with act we get `make: command not found`

### System information

**act version:** 0.2.7
**platform:** Ubuntu 19.10

### How to reproduce:
For example with this simple yaml
```
name: CI
on: [push]
jobs:
  test:
    runs-on: ubuntu-latest
    name: Unit tests
    steps:
      - name: Set up Go 1.14
        uses: actions/setup-go@v1
        with:
          go-version: 1.14
        id: go

      - name: Check out code into the Go module directory
        uses: actions/checkout@v2

      - name: Run unit tests
        run: make test
```

Command executed: `act` 

### Logs:

> [CI/Unit tests] 🚀  Start image=node:12.6-buster-slim
> [CI/Unit tests]   🐳  docker run image=node:12.6-buster-slim entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
> [CI/Unit tests]   🐳  docker cp src=/home/ianaz/workspace/PROJECTNAME/. dst=/github/workspace
> [CI/Unit tests] ⭐  Run Set up Go 1.14
> [CI/Unit tests]   ☁  git clone 'https://github.com/actions/setup-go' # ref=v1
> [CI/Unit tests]   🐳  docker cp src=/home/ianaz/.cache/act/actions-setup-go@v1 dst=/actions/
> [CI/Unit tests]   💬  ##[debug]evaluating 3 versions
> [CI/Unit tests]   💬  ##[debug]matched: 1.14.2
> [CI/Unit tests]   💬  ##[debug]isExplicit: 1.14.2
> [CI/Unit tests]   💬  ##[debug]explicit? true
> [CI/Unit tests]   💬  ##[debug]checking cache: /opt/hostedtoolcache/go/1.14.2/x64
> [CI/Unit tests]   💬  ##[debug]not found
> [CI/Unit tests]   💬  ##[debug]Downloading Go from: https://storage.googleapis.com/golang/go1.14.2.linux-amd64.tar.gz
> [CI/Unit tests]   💬  ##[debug]Downloading https://storage.googleapis.com/golang/go1.14.2.linux-amd64.tar.gz
> [CI/Unit tests]   💬  ##[debug]Downloading /home/actions/temp/63c13e29-3830-4c96-a233-a73a1c9165e3
> [CI/Unit tests]   💬  ##[debug]download complete
> | [command]/bin/tar xzC /home/actions/temp/f5a2b741-e617-4734-bf25-5f4d4121b4fd -f /home/actions/temp/63c13e29-3830-4c96-a233-a73a1c9165e3
> [CI/Unit tests]   💬  ##[debug]Caching tool go 1.14.2 x64
> [CI/Unit tests]   💬  ##[debug]source dir: /home/actions/temp/f5a2b741-e617-4734-bf25-5f4d4121b4fd/go
> [CI/Unit tests]   💬  ##[debug]destination /opt/hostedtoolcache/go/1.14.2/x64
> [CI/Unit tests]   💬  ##[debug]finished caching tool
> [CI/Unit tests]   💬  ##[debug]Go tool is cached under /opt/hostedtoolcache/go/1.14.2/x64
> [CI/Unit tests]   ⚙  ::set-env:: GOROOT=/opt/hostedtoolcache/go/1.14.2/x64
> [CI/Unit tests]   ⚙  ::add-path:: /opt/hostedtoolcache/go/1.14.2/x64/bin
> [CI/Unit tests]   ❓  ##[add-matcher]/actions/actions-setup-go@v1/.github/go.json
> [CI/Unit tests]   ✅  Success - Set up Go 1.14
> [CI/Unit tests] ⭐  Run Check out code into the Go module directory
> [CI/Unit tests]   ✅  Success - Check out code into the Go module directory
> [CI/Unit tests] ⭐  Run Run unit tests
> | /github/workflow/2: line 2: make: command not found
> [CI/Unit tests]   ❌  Failure - Run unit tests
> Error: exit with `FAILURE`: 127****

**Comment by ianaz** [comment]
Sorry, had to read the whole documentation before creating an issue :)

`act -P ubuntu-latest=nektos/act-environments-ubuntu:18.04 -v` seem to fix

**Comment by ethan797979** [comment]
Sorry I don’t want to receive this kind email thank you.



於 星期四, 4月 23, 2020, 20:19，Silvio Rainoldi <notifications@github.com> 寫道：




Closed #206.
